flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/FileSystemConnectorOptions.java

```diff
@@ -245,6 +245,13 @@ public class FileSystemConnectorOptions {
                     .withDescription(
                             "The compaction target file size, the default value is the rolling file size.");
 
+    public static final ConfigOption<Boolean> HIVE_SINK_PARTITION_COMMIT_SUCCESS_FILE_ENABLED =
```

Comment:
It's a hive specifc configuration, should be moved to HiveOptions

Reply:
And I think we also should add a new option like `Hive_SINK_PARTITION_COMMIT_SUCCESS_FILE_NAME` in HiveOptions